src/ResourceManager/Resources/AzureRM.Resources.psd1

```diff
@@ -120,7 +120,9 @@ CmdletsToExport = 'Get-AzureRmProviderOperation', 'Remove-AzureRmRoleAssignment'
                'Set-AzureRmManagedApplicationDefinition', 
                'Remove-AzureRmManagedApplicationDefinition', 
                'Get-AzureRmManagedApplication', 'New-AzureRmManagedApplication', 
-               'Set-AzureRmManagedApplication', 'Remove-AzureRmManagedApplication'
+               'Set-AzureRmManagedApplication', 'Remove-AzureRmManagedApplication',
```

Comment:
@jeffrey-ACE looks like there are a few files in the `Resources` that are being changed that were missed when separating this out into a new module. Would you mind reverting the changes made in this project?

Reply:
@cormacpayne will do